shared/src/persistence/dynamo/workitems/getWorkItemMappingsByDocketNumber.test.js

```diff
@@ -0,0 +1,28 @@
+const client = require('../../dynamodbClientService');
+const {
+  applicationContext,
+} = require('../../../business/test/createTestApplicationContext');
+const {
+  getWorkItemMappingsByDocketNumber,
+} = require('./getWorkItemMappingsByDocketNumber');
+
+describe('getWorkItemMappingsByDocketNumber', () => {
+  beforeEach(() => {
+    client.query = jest.fn();
+  });
+
+  it('should call client.query with pk of case|{docketNumber}', async () => {
+    const mockDocketNumber = '101-21';
+
+    await getWorkItemMappingsByDocketNumber({
+      applicationContext,
+      docketNumber: mockDocketNumber,
+    });
+
+    expect(client.query.mock.calls[0][0]).toMatchObject({
+      ExpressionAttributeValues: {
+        ':pk': `case|${mockDocketNumber}`,
```

Comment:
Seems like we'd also want to include the `KeyConditionExpression` and rest of `ExpressionAttributeValues` as they're what really drive this query.

Reply:
Updated!